GetPlayerInformation.lua

```diff
@@ -737,13 +737,12 @@ local getAuraDuration = function(spellId, unitId)
     --spellId = customBuffDuration or spellId --can't replace the spellId by customBuffDurationSpellId has it wount be found in LIB_OPEN_RAID_PLAYERCOOLDOWNS
 
     if (bIsNewUnitAuraAvailable) then
-        local bBatchCount = false
         local bUsePackedAura = true
         auraSpellID = customBuffDuration or spellId
         auraDurationTime = 0 --reset duration
         auraUnitId = unitId or "player"
 
-        AuraUtil.ForEachAura(auraUnitId, "HELPFUL", bBatchCount, handleBuffAura, bUsePackedAura) --check auras to find a buff for the spellId
+        AuraUtil.ForEachAura(auraUnitId, "HELPFUL", nil, handleBuffAura, bUsePackedAura) --check auras to find a buff for the spellId
```

Comment:
Seems like this was always "incorrect", but in 10.2.5 AuraUtil.ForEachAura started using C_UnitAuras.GetAuraSlots which seems to be more strict about type checking, instead of just assuming that `false` means `0` for this arg.

https://github.com/Gethe/wow-ui-source/blob/45cb43fde1273ac34a6238cd5c1a377013067049/Interface/FrameXML/AuraUtil.lua#L76-L85